src/styles/blocks/page.scss

```diff
@@ -0,0 +1,6 @@
+.page {
+  margin: 0 auto;
+  padding: 50% 0;
+  display: flex;
+  justify-content: center;
```

Comment:
`justify-content: center;` is causing your page to have the center arrangement for the bottom row.

Perhaps use margins to center the calendar horizontally. You also need to think about how to center the calendar vertically.

Reply:
Owkay noted